src/microbe_stage/membrane_system.cpp

```diff
@@ -206,6 +207,25 @@ void
     }
 }
 
+void
+    MembraneComponent::setHealthPercentage(float value)
+{
+    healthPercentage = std::clamp<float>(value, 0.0, 1.0);
```

Comment:
Would this line work like this: `std::clamp(value, 0.0f, 1.0f);` (all floats, so template type deduction should work)